### PR TITLE
Add cache headers to uploads, upload index.html separately

### PR DESCRIPTION
**Pull request by zwolf**
Staging branch URL: https://pr-{NUMBER}.lab-preview.zooniverse.org/

Fixes # .

Describe your changes.

# Review Checklist

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [ ] Are the tests passing locally and on Travis?

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -30,8 +30,17 @@ jobs:
...
         creds: ${{ secrets.AZURE_STATIC_SITES }}
         inlineScript: |
-            az storage blob upload-batch --account-name zooniversestatic -d '$web/lab.zooniverse.org' -s ./dist
-
+            az storage blob upload \
+              --account-name zooniversestatic \
+              --content-cache-control 'max-age=60' \
+              --container-name '$web' \
+              --name 'lab.zooniverse.org/index.html' \
+              --file ./index.html
```
Comment (srallen) [inline .github/workflows/main.yml:38]: I think this has to be ./dist/index.html

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -30,8 +30,17 @@ jobs:
...
+            az storage blob upload-batch \
+              --account-name zooniversestatic \
+              --content-cache-control 'public, max-age=604800, immutable' \
+              -destination '$web/lab.zooniverse.org' \
+              -source ./dist
```
Comment (srallen) [inline .github/workflows/main.yml:43]: I think this will upload the index.html file again with different cache control headers.

**Review by srallen — CHANGES_REQUESTED** [review]
The output html file is also in the dist folder, so I think the upload of the html file needs to be adjusted. If it would help make this easier, we could consider modifying the webpack out config more to make this feasible.

File: .github/workflows/main.yml (hunk 2 above)
Comment (zwolf) [reply .github/workflows/main.yml:37]: ✔️

File: .github/workflows/main.yml (hunk 1 above)
Comment (zwolf) [reply .github/workflows/main.yml]: ✔️

**Review by srallen — APPROVED** [review]
LGTM. Let's try it out.

**Pull request merged**